### ISSUE TITLE
[REQUEST] Flip-scan with D'

### ISSUE THREAD

**Issue by JoniColeman** [chrchang/plink-ng#58]
**tl;dr - it would be useful if there were an option for using D' instead of correlation for --flip-scan.** 

--flip-scan is a useful tool in quality control for identifying flipped ambiguous SNPs when component parts of a GWAS have been genotyped separately. It can also be used to identify variants that have been miscalled during genotyping, as they (may) depart from the expected pattern of LD with neighbouring SNPs. To some extent, this is captured by --flip-scan's current correlation-based measure. However, D' might be a more useful measure here, as it is more sensitive to the presence of unlikely haplotypes. As such, it would be useful if --flip-scan had a D' mode as well. 

Ideally, the output would be a variant of that from --flip-scan verbose

```
CHR_INDX     Chromosome code
SNP_INDX     Index variant identifier
BP_INDX      Index variant base-pair coordinate
A1_INDX      Index variant allele 1
SNP_PAIR     Second variant identifier
BP_PAIR      Second variant base-pair coordinate
A1_PAIR      Second variant allele 1
D_PRIME_A    Case-only D'
D_PRIME_U    Control-only D'

```

**Comment by BenjaminDEMAILLE** [comment]
#355 implements the base `--flip-scan` command in plink2, which this request builds on. It is correlation-based, like 1.9's, so it does not answer the D' part; but it is the piece that has to exist first, and adding a D' mode on top of it is then a matter of swapping the per-pair statistic — the neighborhood walk, the case/control split and the reporting are all in place.

Verified field-for-field against PLINK 1.9 on a simulated set with deliberately flipped variants and on a 1000 Genomes chr21 subset, and 4.4-6.6x faster than 1.9 on 5000 samples x 50000 variants.
